.github/workflows/build.yml

```diff
@@ -0,0 +1,42 @@
+name: Build
+
+on: 
+  push:
+    branches:
+      - master
+      - release-*
+    tags:
+      - v*
+    
+  pull_request:
+    branches: 
+    - master
+    - release-*
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    env:
+      JDK_VER: 13.0.x
+      DAPR_RUNTIME_VER: 0.3.0
+    steps:
+    - uses: actions/checkout@v1
+    - name: Set up ${{ env.JDK_VER }}
+      uses: actions/setup-java@v1
+      with:
+        java-version: ${{ env.JDK_VER }}
+    - name: Set up Dapr CLI
+      run: wget -q https://raw.githubusercontent.com/dapr/cli/master/install/install.sh -O - | /bin/bash
```

Comment:
We will need to change it to build dapr edge since our release must work with the 0.4 milestone. The last release is 0.3, so this will break as soon as we make our API compatible with 0.4.